app/blueprints/post_blueprint.rb

```diff
@@ -10,6 +10,8 @@ class PostBlueprint < Blueprinter::Base
       to_return = FacebookPostBlueprint.render_as_hash(post)
     when "Zorki::Post"
       to_return = InstagramPostBlueprint.render_as_hash(post)
+    when "YoutubeArchiver::Video"
+      to_return = YoutubeVideoBlueprint.render_as_hash(post)
```

Comment:
Any chance we can rename this to "YouTube" or was this for a specific reason? (name spacing?

Reply:
Oops. To line up with Zenodotus, this should read `YoutubePostBlueprint`, not `YoutubeVideoBlueprint`. Are you suggesting `YoutubeBlueprint` instead? Or suggesting renaming the entire scraping module `Youtube`?

Reply:
YouTubePostBlueprint (capital 't')

Reply:
Resolved per [slack discussion](https://techandcheck.slack.com/archives/CHQAH5FGA/p1649101741310739)